### PR TITLE
Custimizable mind state for priority skills

### PR DESCRIPTION
**Pull request by skyleronken**
Can define the mind state at which training-manager will now trigger the
hunting cycle. Issue #1287.

### CODE REVIEW

**Comment by skyleronken** [comment]
Example yaml:

priority_skills_lower_limit: 4

File: training-manager.lic [hunk 1]
```diff
@@ -48,7 +48,8 @@ class TrainingManager
   end
 
   def priority_skills_low?
-    @settings.training_manager_priority_skills.any? { |skill| DRSkill.getxp(skill) <= 6 }
+    lower_limit = (@settings.priority_skills_lower_limit || 6)
```
Comment (rpherbig) [inline training-manager.lic]: This default value should be in `base.yaml` instead of ||ed here.

**Comment by skyleronken** [comment]
Good call. Those changes should be better.

File: training-manager.lic (hunk 1 above)
Comment (skyleronken) [inline training-manager.lic]: Good call! Change mades!

**Pull request merged**